src/main/java/com/snowflake/kafka/connector/internal/BufferThreshold.java

```diff
@@ -24,108 +23,100 @@ public abstract class BufferThreshold {
-   * @param currentBufferSizeInBytes current size of buffer in bytes
-   * @return true if bytes threshold has reached.
+   * @param currBufferByteSize current size of buffer in bytes
+   * @return true if the currByteSize > configByteSizeThreshold
    */
-  public boolean isFlushBufferedBytesBased(final long currentBufferSizeInBytes) {
-    return currentBufferSizeInBytes >= bufferSizeThresholdBytes;
+  public boolean shouldFlushOnBufferByteSize(final long currBufferByteSize) {
+    return currBufferByteSize >= bufferByteSizeThreshold;
   }
 
   /**
-   * Returns true if number of ({@link SinkRecord})s in current buffer is more than threshold
-   * provided.
+   * Returns true the buffer should flush based on the current buffer record count
    *
    * <p>Threshold is config parameter: {@link
    * com.snowflake.kafka.connector.SnowflakeSinkConnectorConfig#BUFFER_COUNT_RECORDS}
    *
-   * @param currentBufferedRecordCount current size of buffer in terms of number of kafka records
-   * @return true if number of kafka threshold has reached in buffer.
+   * @param currentBufferedRecordCount current size of buffer in number of kafka records
+   * @return true if the currRecordCount > configRecordCountThreshold
    */
-  public boolean isFlushBufferedRecordCountBased(final long currentBufferedRecordCount) {
+  public boolean shouldFlushOnBufferRecordCount(final long currentBufferedRecordCount) {
     return currentBufferedRecordCount != 0
-        && currentBufferedRecordCount >= bufferKafkaRecordCountThreshold;
+        && currentBufferedRecordCount >= bufferRecordCountThreshold;
   }
 
   /**
-   * If difference between current time and previous flush time is more than threshold return true.
+   * Returns true the buffer should flush based on the last flush time
    *
-   * @param previousFlushTimeStampMs when were previous buffered records were flushed into internal
-   *     stage for snowpipe based implementation or previous buffered records were sent in
-   *     insertRows API of Streaming Snowpipe.
-   * @return true if time based threshold has reached.
+   * <p>Threshold is config parameter: {@link
+   * com.snowflake.kafka.connector.SnowflakeSinkConnectorConfig#BUFFER_FLUSH_TIME_SEC}
+   *
+   * @param previousFlushTimeStampMs when the previous buffered records flushed
+   * @return true if currentTime - previousTime > configTimeThreshold
    */
-  public boolean isFlushTimeBased(final long previousFlushTimeStampMs) {
+  public boolean shouldFlushOnBufferTime(final long previousFlushTimeStampMs) {
     final long currentTimeMs = System.currentTimeMillis();
     return (currentTimeMs - previousFlushTimeStampMs)
-        >= (this.flushTimeThresholdSeconds * SECOND_TO_MILLIS);
+        >= (this.bufferFlushTimeThreshold * SECOND_TO_MILLIS);
   }
 
   /** @return Get flush time threshold in seconds */
   public long getFlushTimeThresholdSeconds() {
-    return flushTimeThresholdSeconds;
+    return this.bufferFlushTimeThreshold;
   }
 
-  /**
```

Comment:
why are we removing this?
public methods are recommended to have javadocs

Reply:
oh good call out, removed by mistake on my end